types/assert-plus/index.d.ts

```diff
@@ -5,103 +5,106 @@
 
 /// <reference types="node" />
 
+type Func = (...args: any[]) => any;
+export {};
```

Comment:
I actually don't think you need this particular line, since this file contains explicit exports and so `Func` won't be exported, right? @peterblazejewicz 

(You probably want to move this below the import, though.)

Reply:
If I remote that I get 
```
ERROR: 8:6  strict-export-declare-modifiers  All declarations in this module are exported automatically. Prefer to explicitly write 'export' for clarity. If you have a good reason not to export this declaration, add 'export {}' to the module to shut off automatic exporting. See: https://github.com/microsoft/DefinitelyTyped-tools/blob/master/packages/dtslint/docs/strict-export-declare-modifiers.md
```
so I guess I could export the type, but I don't really want to...

Reply:
Darn, I guess d.ts rules are slightly different than regular ts... :D